src/test/t-io.c

```diff
@@ -590,6 +602,22 @@ TEST_FUNCTION_START(flint_fprintf, state)
         ARB_POLY_SET(xarb_poly);
         ACB_POLY_SET(xacb_poly);
 
+        gr_ctx_t xgr_ctx;
+        gr_ctx_init_fmpzi(xgr_ctx);
+        gr_ptr xgr = gr_heap_init(xgr_ctx);
+        gr_ptr xgr_vec = gr_heap_init_vec(GR_VEC_LEN, xgr_ctx);
+        gr_poly_t xgr_poly;
+        gr_mat_t xgr_mat;
+        GR_IGNORE(gr_set_ui(xgr, 5, xgr_ctx));
+        GR_IGNORE(gr_set_ui(GR_ENTRY(xgr_vec, 0, xgr_ctx->sizeof_elem), 1, xgr_ctx));
+        GR_IGNORE(gr_set_ui(GR_ENTRY(xgr_vec, 1, xgr_ctx->sizeof_elem), 2, xgr_ctx));
+        GR_IGNORE(gr_set_str(GR_ENTRY(xgr_vec, 2, xgr_ctx->sizeof_elem), "I", xgr_ctx));
+        gr_poly_init(xgr_poly, xgr_ctx);
+        GR_IGNORE(gr_poly_set_coeff_si(xgr_poly, 1, -1, xgr_ctx));
+        gr_mat_init(xgr_mat, 1, 2, xgr_ctx);
+        GR_IGNORE(gr_set_ui(gr_mat_entry_ptr(xgr_mat, 0, 0, xgr_ctx), 4, xgr_ctx));
+        GR_IGNORE(gr_set_ui(gr_mat_entry_ptr(xgr_mat, 0, 1, xgr_ctx), 3, xgr_ctx));
```

Comment:
Before any merging, please initialize, set and clear as done with all other variables.